tmt-app/src/Components/NavBar.jsx

```diff
@@ -0,0 +1,171 @@
+          <Link key="Homepage" to="/" component={RouterLink}>
+            <img src={BTLogo} alt="logo" />
+          </Link>
+          <Box
+            sx={{
+              display: {
+                xs: 'flex',
+                md: 'none',
+              },
+            }}
+          >
+            <IconButton
+              aria-label="options"
+              color="black"
+              onClick={handleOpenNavMenu}
+            >
+              <MenuIcon />
+            </IconButton>
+            <Drawer
+              anchor="right"
+              open={navMenu}
+              onClose={handleCloseNavMenu}
+              ModalProps={{ disableScrollLock: 'true' }}
+              sx={{
+                display: {
+                  xs: 'flex',
+                  md: 'none',
+                },
+              }}
+            >
+              <Box>
+                <ListItem sx={{ justifyContent: 'flex-end' }}>
+                  <IconButton
+                    aria-label="options"
+                    color="black"
+                    onClick={handleCloseNavMenu}
+                  >
+                    <CloseIcon />
+                  </IconButton>
+                </ListItem>
+                {tabs.map((tab, id) => {
+                  return (
+                    <List key={`${tab}-{id}`}>
+                      <ListItem disablePadding onClick={handleCloseNavMenu}>
+                        <ListItemButton>
+                          <Link
+                            key={id}
+                            to={tab.path}
+                            component={RouterLink}
+                            underline="none"
+                            sx={{
+                              margin: '0 1rem',
+                              color: 'Boba.main',
+                            }}
+                          >
+                            <Typography variant="h5">{tab.name}</Typography>
+                          </Link>
+                        </ListItemButton>
+                      </ListItem>
+                    </List>
+                  );
+                })}
+              </Box>
+            </Drawer>
+          </Box>
+          <Box
+            sx={{
+              flexGrow: 1,
+              display: { xs: 'none', md: 'flex' },
+              justifyContent: 'flex-end',
+              alignItems: 'center',
+            }}
+          >
+            {tabs.map((tab, id) => {
+              return id === tabs.length - 1 ? (
+                <Link
+                  key={id}
+                  to={tab.path}
+                  component={RouterLink}
+                  underline="none"
+                  sx={{
+                    color: 'white',
+                    margin: '0 1rem',
+                    padding: '0.5rem 3rem',
+                    backgroundColor: 'black',
+                    borderRadius: '30px',
+                    ':hover': {
+                      backgroundColor: '#484848',
```

Comment:
We should pull all colors from theme, not hard code them

![image](https://github.com/BobaTalks/pick-a-major/assets/12107969/2d24cd81-20bd-4dea-a9b2-bdbfb992548e)
This also doesn't match design spec. can we take a little extra time reviewing the designs as part of the development process?

![image](https://github.com/BobaTalks/pick-a-major/assets/12107969/eb529cea-dc42-4ff2-b836-3e0260c485f7)

Reply:
This does not relatively match the design because the nav bar will be too big if we are going by the spec of 123px for the height. I am also using the default max-width from mui to keep the design in-place. Any screen size bigger than the max-width would just add on more empty space to the left and right side.